movie-browser/src/App.js

```diff
@@ -1,31 +1,61 @@
 /* eslint-disable no-unused-vars */
 import React from 'react'
-import CardContainer from './CardContainer'
+import CardContainer from './view/CardContainer'
+import FeaturedMovie from './view/FeaturedMovie'
 import ApiReader from './model/ApiReader';
 
 // *** constant(s)
 const url = 'https://ghibliapi.herokuapp.com/films'
+const btnTxtDef = "View all"
```

Comment:
no need to store this in a variable. And even if you did, please dont use these types of names. Names must be as explicit as possible

Reply:
Removed. There is quite a bit of story involved but I'm not gonna tell everything.
It can be compressed as : I thought too much.